### PR TITLE
PD-115 Create Update HA SCALE Article

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit creates the new UpdateHASCALE.md article, makes a minor change in UpdateSCALE.md by adding a description to the front matter.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md [hunk 1]
```diff
@@ -30,13 +31,12 @@ Using [Active Directory]({{< relref "/content/SCALE/SCALEUIReference/Credentials
...
 
-![LocalUsersSCALE](/images/SCALE/22.12/LocalUsersSCALE.png "List of Local User Accounts") 
+![AllUsersScreenSCALE](/images/SCALE/22.12/AllUsersScreenSCALE.png "Local User non-Built-in Accounts") 
 
 TrueNAS hides all built-in users (except root) by default. Click the toggle **Show Built-In Users** to see all built-in users.
 
 ## Creating an Admin User Account
-SCALE has implemented rootless log in. All systems should create and begin using an admin user as a replacement for the root user. 
-A system warning alert displays until you create the admin user. 
+SCALE has implemented rootless log in. All systems should create and begin using an admin user as a replacement for the root user.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md:39]:
```suggestion
SCALE has implemented rootless login. All systems should create and use an admin user to replace the root user.  
```

File: content/SCALE/SCALETutorials/SystemSettings/UpdateHASCALE.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+3. Select the update file and start the process. 
+   Click **Choose File** and select the <kbd>update</kbd> file downloaded to your system, then click **Apply Update** to start the update process.
+   After the system to finishes updating it reboots.
+
+4. Sign into the SCALE UI. If using root to sign in, create the admin account now.
+   If using admin, continue to the next step.
+
+5. Verify the system license after the update. Go to **System Settings > General**.
+   Verify the license information in screenshot of the **Support** widget taken before the update matches the information on the **Support** widget after updating the system.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/SystemSettings/UpdateHASCALE.md:41]:
```suggestion
   Verify the license information in the screenshot of the **Support** widget you took before the update matches the information on the **Support** widget after updating the system.  
```

File: content/SCALE/SCALETutorials/SystemSettings/UpdateHASCALE.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+   Verify the license information in screenshot of the **Support** widget taken before the update matches the information on the **Support** widget after updating the system.  
+
+6. Verify the admin user settings, or if not created, [create the admin user]({{< relref "ManageLocalUsersSCALE.md" >}}) account now. 
+   Sudo settings changed in 22.0.1. If you want the admin account to have the ability to execute sudo commands in an SSH session, select the option for the sudo access you want to allow. 
+   Also verify **Shell** is set to **bash** if you want the admin user to have the ability to execute commands in **Shell**. 
+   To set a location where the admin user can save to, browse to and select the dataset in **Home Directory**. If set to the default **/nonexistent** files are not saved for this user.
+
+7. Verify the SSH service settings. Go to **System Settings > Services**, then edit the **SSH** service. 
+   If you just created the admin user, select **Log In As Admin with Password** and clear the **Log In As Root with Password** checkbox. Click **Save**. Select **Start automatically**, which means this service automatically stars after a system reboot.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/SystemSettings/UpdateHASCALE.md:49]:
```suggestion
6. Verify the admin user settings, or if not created, [create the admin user]({{< relref "ManageLocalUsersSCALE.md" >}}) account now. 
   Sudo settings changed in 22.0.1. If you want the admin account to have the ability to execute sudo commands in an SSH session, select the option for the sudo access you want to allow. 
   Also, verify **Shell** is set to **bash** if you want the admin user to have the ability to execute commands in **Shell**. 
   To set a location where the admin user can save to, browse to, and select the dataset in **Home Directory**. If set to the default **/nonexistent** files are not saved for this user.

7. Verify the SSH service settings. Go to **System Settings > Services**, then edit the **SSH** service. 
   If you just created the admin user, select **Log In As Admin with Password** and clear the **Log In As Root with Password** checkbox. Click **Save**. Select **Start automatically**, which means this service automatically starts after a system reboot.
```

File: content/SCALE/SCALETutorials/SystemSettings/UpdateHASCALE.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+8. Test the admin user access to the UI and to SSH if you just created it. 
+   a. Log out of the UI. 
+   b. Enter the admin user credentials in the sign in splash screen. 
+   c. Start an SSH session and log into it with the admin credentials. 
+   d. Test access by issuing the `zectl list` command. 
+      This should display the list of boot environments. If nothing happens, try `sudo zectl list` and enter the admin password.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/SystemSettings/UpdateHASCALE.md:56]:
```suggestion
8. Test the admin user access to the UI and to SSH if you just created it. 
   a. Log out of the UI. 
   b. Enter the admin user credentials in the sign-in splash screen. 
   c. Start an SSH session and log into it with the admin credentials. 
   d. Test access by issuing the `zectl list` command. 
      This should display the list of boot environments. If nothing happens, try `sudo zectl list` and enter the admin password. 
```

**Review by tonyriv3 — CHANGES_REQUESTED** [review]

File: content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md (hunk 1 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md:39]: Claire, I'm not sure how your article got into my PR. Would you review this open comment and either accept or change the suggestion made by Tony.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**